packaging/nuget/release.py

```diff
@@ -76,10 +76,8 @@
 
     print('')
 
-    if len(pkgfiles) > 0:
-        print('Created package:')
-        for pkg in pkgfiles:
-            print(pkg)
-    else:
-        print('Package not created.')
+    if not p.verify(pkgfile):
```

Comment:
Did you verify that verify works with the new package builds?

Reply:
yes. note also that the package contents are different from before.